content/it/docs/concepts/cluster-administration/federation.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Federation
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+{{< include "federation-current-state.md" >}}
```

Comment:
Because of the reformatting of the federation page, this needs to be updated as well. Can you update this to:
```suggestion
{{< deprecationfilewarning >}}
{{< include "federation-deprecation-warning-note.md" >}}
{{< /deprecationfilewarning >}} 
```

And copy and translate the `content/en/includes/federation-deprecation-warning-note.md` file to `content/it/includes/federation-deprecation-warning-note.md`